docs/manual/docs/sample-scripts/spo/add-custom-clientside-webpart-to-modern-page.md

```diff
@@ -0,0 +1,53 @@
+# Add custom client-side WebPart to modern page
```

Comment:
We should reference pages from `mkdocs.yml` or they won't included in the generated docs :)

Reply:
Good to know for next time :)